src/Ddeboer/DataImport/Filter/DateTimeThresholdFilter.php

```diff
@@ -47,14 +47,14 @@ public function __construct(
     /**
      * {@inheritDoc}
      */
-    public function filter(array $item)
+    public function __invoke(array $item)
     {
         if ($this->threshold == null) {
             throw new \LogicException('Make sure you set a threshold');
```

Comment:
I don't know what was exactly the reason, i'm not the author of this. But you are right, it should be a constructor argument.